src/Peachpie.Library.Network/CURLConstants.cs

```diff
@@ -1067,11 +1086,88 @@ public override void Apply(Context ctx, HttpWebRequest request)
 
         private static void LoadCookieFile(HttpWebRequest request, PhpStream stream)
         {
-            // TODO: load netscape-like or header-style cookies from stream
+            // load netscape-like or header-style cookies from stream
 
-            // request.CookieContainer.Add() // add parsed cookie
+            // TODO: add support for header-style cookies (`Set-Cookie: ...`).
+            //       currently, only curl's netscape-like cookies are supported.
 
             // request.CookieContainer.SetCookies( ... ) // set header style cookies
+
+            string line = String.Empty;
+
+            // iterate over the cookie jar lines
+            while (!stream.Eof)
+            {
+
+                // read a line, and strip off the possible end-of-line characters
+                line = stream.ReadLine(-1, null).Trim();
+
+                // ignore blank lines
+                if (String.IsNullOrWhiteSpace(line))
```

Comment:
won't be a whitespace :) can be changed to `IsNullOrEmpty()`

Reply:
@jakubmisek Force of habit :D Even though I `.Trim()` it just one line above